Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -557,6 +576,71 @@ - (void)sectionEditorFinishedEditing:(SectionEditorViewController*)sectionEditor
     [self fetchArticle];
 }
 
+
+
+
+
+
+
+
+
+
+
+
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController*)previewingContext:(id<UIViewControllerPreviewing>)previewingContext
+                      viewControllerForLocation:(CGPoint)location {
+
+//return
+//[[WMFArticleContainerViewController alloc] initWithArticleTitle:[[MWKTitle alloc] initWithURL:[NSURL URLWithString:@"http://en.wikipedia.org/wiki/Cat"]]
```

Comment:
If I do this it doesn't get dealloc'ed - although it only works if you press on a non-link... beginning to think there js anchor touch events are in the mix...